engine/classes/ElggPluginPackage.php

```diff
@@ -308,7 +311,7 @@ private function hasSaneDependencies() {
 
 						if ($version_compare) {
 							$this->errorMsg =
-								_elgg_services()->translator->translate('ElggPluginPackage:InvalidPlugin:CircularDep',
+								elgg_echo('ElggPluginPackage:InvalidPlugin:CircularDep',
```

Comment:
Is it possible to keep the `elgg_echo` on the same line a `$this->errorMsg`? now that the line is shorter.....